doc/cutting-a-release.md

```diff
@@ -81,41 +89,54 @@ git commit -m"Create ${RELEASE}.x release branch" .
 git push --set-upstream origin ${RELEASE}.x
 ```
 
+NOTE: No code review or Pull Request is needed as part of this step.
+
 ## Update the documentation links
 
-Pushing the branch should start the CI builds. One of the CI builds
-automatically pushes a new version of the documentation to the `gh-pages`
-branch. You should checkout that branch, change the `index.html` page to link to
-this new version and commit it:
+Pushing the branch should start the CI builds. You can find and watch the
+triggered builds at https://travis-ci.com/googleapis/google-cloud-cpp. One of
+the CI builds automatically pushes a new version of the documentation to the
+`gh-pages` branch. Wait for this build to complete, then you should checkout
+`gh-pages` and change the `index.html` page to link to this new version and
+commit it:
 
 ```bash
-git pull
 git checkout gh-pages
-sed -i "s;\(.*URL='\).*\(/index.html.*\);\1${RELEASE}\2;" index.html 
-git commit -m"Update documentation to ${RELEASE}" .
+git pull
+```
+
+Change the value of the meta tag's `URL=` parameter to refer to the
+`index.html` file in the directory matching the new release version number.
+
+```
+git commit -am "Update documentation to ${RELEASE}"
 git push
 ```
 
+You are now finished with this "releases" clone of the repo that we created in
+the instructions above. You may now remove this directory.
+
 ## Bump the version numbers in `master`
 
-Working in your fork of `gooogle-cloud-cpp`: bump the version numbers, and send
-the PR for review against `master`. For an example, look at
-[#1375](https://github.com/googleapis/google-cloud-cpp/pull/1375).
+Working in your fork of `gooogle-cloud-cpp`: bump the version numbers to the
+*next* version (i.e., one version past the release you just did above), and
+send the PR for review against `master`. For an example, look at
+[#1962](https://github.com/googleapis/google-cloud-cpp/pull/1962)
 
 ## Create a pre-release tag
 
 Create a pre-release using
 [GitHub](https://github.com/googleapis/google-cloud-cpp/releases/new).
-Make sure your reference the `v0.N.x` branch, and you check the `pre-release`
-checkbox.
+Make sure you reference the `v0.N.x` branch, set a tag name like `v0.N.0-pre1`,
+and you check the `pre-release` checkbox.
```

Comment:
should that be "and check the `pre-release` checkbox."?

Reply:
Yep.